js/src/components/shipping-rate-section/minimum-order-card/groupShippingRatesByFreeShippingThreshold.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef { import(".~/data/actions").ShippingRate } ShippingRate
+ * @typedef { import("./typedefs").MinimumOrderGroup } MinimumOrderGroup
+ */
+
+/**
+ * Group shipping rates by free shipping threshold into minimum order groups.
+ *
+ * @param {Array<ShippingRate>} shippingRates Array of shipping rates.
+ * @return {Array<MinimumOrderGroup>} Array of minimum order groups.
+ */
+const groupShippingRatesByFreeShippingThreshold = ( shippingRates ) => {
+	const map = new Map();
+
+	shippingRates.forEach( ( shippingRate ) => {
+		const {
+			options: { free_shipping_threshold: threshold },
+			currency,
+		} = shippingRate;
+
+		const thresholdCurrency = `${ threshold } ${ currency }`;
```

Comment:
💅  Same as https://github.com/woocommerce/google-listings-and-ads/pull/1445#discussion_r879024509

Reply:
Addressed in [`36298a2` (#1445)](https://github.com/woocommerce/google-listings-and-ads/pull/1445/commits/36298a2f109982916bac3bd219133efe4384c252).